src/pyproject_fmt/formatter/project.py

```diff
@@ -19,7 +19,7 @@
 
     from .config import Config
 
-_PY_MIN_VERSION: int = 7
+_PY_MIN_VERSION: int = 0
```

Comment:
Wait this should still be 7, to handle when requires-python is not set

Reply:
Apologies that we haven't been able to find another fix for this. We've elaborated about it at https://github.com/tox-dev/pyproject-fmt/issues/137#issuecomment-1777775742.

Reply:
Moving the comment here instead:

To elaborate a bit why the fix is not 100% solid, other than what has been outlined in the PR description:

1. The default value of `_PY_MIN_VERSION` is already used at [L67](https://github.com/tox-dev/pyproject-fmt/blob/1.2.0/src/pyproject_fmt/formatter/project.py#L67).
2. Then, it is also used at [L76](https://github.com/tox-dev/pyproject-fmt/blob/1.2.0/src/pyproject_fmt/formatter/project.py#L76), which is probably not intended. Instead, it should be derived from the parsed `SpecifierSet` in `specifiers`, which already resolved the `requires-python` setting, but differently than it is done right now, also considering "minimum versions" instead of just resolving the `max_version_specifier`.

Reply:
I don't I can accept setting this min version with 0 🤔 we'll ned the more elaborate variant 🤔

Reply:
Hi Bernát. Let's have another look at the adjusted variant of this patch, not modifying `_PY_MIN_VERSION` now? Please let me know if you think `_get_max_version_tox` will need a corresponding adjustment, or any other suggestions to improve the code. Thanks!